osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -47,12 +47,12 @@ public IReadOnlyList<BreakPeriod> Breaks
         private readonly RemainingTimeCounter remainingTimeCounter;
         private readonly BreakArrows breakArrows;
 
+        public readonly BreakInfo Info;
```

Comment:
Rather than exposing this, please use `ChildrenOfType<BreakInfo>.Single()` in the test (check other tests for examples).